tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -13065,16 +13440,51 @@ node_modules/chrome-devtools-frontend/front_end/ui/UIUtils.js(1317,27): error TS
 node_modules/chrome-devtools-frontend/front_end/ui/UIUtils.js(1318,24): error TS2339: Property '_constructor' does not exist on type 'typeof CheckboxLabel'.
 node_modules/chrome-devtools-frontend/front_end/ui/UIUtils.js(1319,74): error TS2339: Property '_constructor' does not exist on type 'typeof CheckboxLabel'.
 node_modules/chrome-devtools-frontend/front_end/ui/UIUtils.js(1324,29): error TS2339: Property 'createChild' does not exist on type 'Element'.
+node_modules/chrome-devtools-frontend/front_end/ui/UIUtils.js(1334,10): error TS2339: Property 'checkboxElement' does not exist on type 'Element'.
```

Comment:
we're now checking `this` and the function has an incorrect `@this {Element}` annotation.